src/sentry/api/endpoints/project_platforms.py

```diff
@@ -11,8 +11,22 @@ class ProjectPlatformSerializer(Serializer):
     def serialize(self, obj, attrs, user):
         return {'platform': obj.platform, 'dateCreated': obj.date_added}
 
+ERR_FIELD_REQUIRED = 'This field is required.'
+
 
 class ProjectPlatformsEndpoint(ProjectEndpoint):
     def get(self, request, project):
         queryset = ProjectPlatform.objects.filter(project_id=project.id)
         return Response(serialize(list(queryset), request.user))
+
+    def put(self, request, project):
+        platform_name = request.DATA.get('platform')
+
+        if platform_name is None:
+            return Response({
+                'errors': {'platform': ERR_FIELD_REQUIRED},
+            }, status=400
+            )
+
+        platform = ProjectPlatform.objects.create(project_id=project.id, platform=platform_name)
```

Comment:
This will generate a 500 if it already exists. Should change this use `ProjectPlatform.objects.get_or_create`.

Reply:
No longer relevant